resources/VERSION

```diff
@@ -0,0 +1 @@
+v0.2.2-59-ga88853a
```

Comment:
It is temporarily added in some of intermediate commits, but it isnt present at the end. Please do not tell me to rebase intermediate commits, I had enough fun with that yesterday...